packages/react-native-client/ios/Utils/PermissionUtils.swift

```diff
@@ -0,0 +1,16 @@
+import AVFoundation
+
+class PermissionUtils {
```

Comment:
**suggestion:** 
Maybe create extension on `AVCaptureDevice` instead of seperate class?

Reply:
I did it that way so that API is somewhat consistent with Android. It's easier to find corresponding code then.

Reply:
I agree.